src/app/shared/components/fy-location/fy-location-modal/fy-location-modal.component.ts

```
@@ -80,14 +85,24 @@
~    }
   }
 
   async checkPermissionStatus() {
```

ESLint

[error] Missing return type on function.

(@typescript-eslint/explicit-function-return-type)